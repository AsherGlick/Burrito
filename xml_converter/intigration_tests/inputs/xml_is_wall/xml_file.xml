<OverlayData>
	<MarkerCategory DisplayName="My Category" Name="mycategory">
	</MarkerCategory>

	<POIs>
		<Trail IsWall="true" Type="mycategory" MapID="50" />
		<Trail IsWall="1" Type="mycategory" MapID="50" />
		<Trail IsWall="false" Type="mycategory" MapID="50" />
		<Trail IsWall="0" Type="mycategory" MapID="50" />
	</POIs>
</OverlayData>
