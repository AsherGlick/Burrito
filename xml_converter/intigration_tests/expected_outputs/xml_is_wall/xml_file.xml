<OverlayData>
<MarkerCategory  DisplayName="My Category" Name="mycategory">
</MarkerCategory>

<POIs>
<Trail  Type="mycategory" IsWall="true" MapID="50"/>
<Trail  Type="mycategory" IsWall="true" MapID="50"/>
<Trail  Type="mycategory" IsWall="false" MapID="50"/>
<Trail  Type="mycategory" IsWall="false" MapID="50"/>
</POIs>
</OverlayData>
